src/pymor/operators/block.py

```diff
@@ -139,7 +139,8 @@ def process_row(row, space):
                     R.append(op.as_range_array(mu))
             return R
 
-        blocks = [process_row(row, space) for row, space in zip(self._blocks, self.range.subspaces)]
+        subspaces = self.range.subspaces if self.blocked_range else self.range
```

Comment:
`subspaces` should be a sequence of vector spaces, right? So, maybe
```suggestion
        subspaces = self.range.subspaces if self.blocked_range else (self.range,)
```
instead?

Reply:
Right, this is what I inteded to write ...